### DISCUSSION TITLE
2022/12/04/OurFourSins

### DISCUSSION THREAD

**Discussion by giscus** [KMIT-Students/blog-demo#12] [General]
# 2022/12/04/OurFourSins

In the end, after all, were you just a sweet dream? Did you even exist in my life, or have I in yours? Chance happened to bring us both together But, sadly, Fate had different plans to scheme My poetry seems to remind me of you too much So, what was I supposed to do? Weep over my ink that only bled your name, to erase you Or hope that Luck gambles in the air between us Only to make sure Fortune intervenes - By Siri

https://kmit-anubhav.github.io/2022/12/04/OurFourSins.html

**Comment by MaruthiCharan2403** [comment]
GOOD ONE

**Comment by creator318** [comment]
Well written.
